src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue

```diff
@@ -74,6 +74,18 @@
         </a>
       </foreignObject>
     </svg>
+
+    <div>
+      <div v-if="hasPrice">
+        {{ tempNFTPrice }} $LIKE
+      </div>
+      <a
+        :href="tempLink"
+        target="_blank"
+      >
+        {{ tempCTA }}
```

Comment:
Please format the `tempCTA` and `tempLink` in this component using `computed` according to `NFTCurrentPrice` instead of passing as prop